### PR TITLE
chore: update docker/git/pettier ignore files 

### PR DESCRIPTION
**Pull request by Nasfame**
ignore: for docker, prettier, eslint

### CODE REVIEW

**Comment by Jolg42** [comment]
Hi @Nasfame 

What are you trying to do? 
I have some guesses based on the changes, though a small description would help us review this.

File: .dockerignore [hunk 1]
```diff
@@ -1,2 +1,6 @@
 node_modules
-packages/**/node_modules
\ No newline at end of file
+packages/**/node_modules
+.idea/
+.vscode/w
```
Comment (Jolg42) [inline .dockerignore:4]:
```suggestion
.vscode/
```
This looked like a typo

File: .prettierignore [hunk 2]
```diff
@@ -9,4 +9,7 @@ dist/
 /packages/client/declaration
 /packages/client/scripts/default-index.js
 /reproductions
-.cache
\ No newline at end of file
+.cache
+node_modules/
```
Comment (Jolg42) [inline .prettierignore:13]: `node_modules/` is not really needed because it's already ignored by default by Prettier

File: .prettierignore [hunk 3]
```diff
@@ -9,4 +9,7 @@ dist/
 /packages/client/declaration
 /packages/client/scripts/default-index.js
 /reproductions
-.cache
\ No newline at end of file
+.cache
```
Comment (Jolg42) [inline .prettierignore:12]: What is this `.cache` directory?

File: .eslintignore [hunk 4]
```diff
@@ -22,3 +22,6 @@ packages/cli/install/
 packages/cli/preinstall/
 packages/cli/**/tmp-*
 reproductions/
+
+
+node_modules/
```
Comment (Jolg42) [inline .eslintignore:27]: Not needed as already ignored by default https://eslint.org/docs/latest/use/configure/ignore

**Review by Jolg42 — APPROVED** [review]
Thanks!

**Comment by Nasfame** [comment]
Thanks @Jolg42

**Comment by Nasfame** [comment]
I ill make sure to add the descriptions next time

**Pull request merged**